Jenkinsfile

```diff
@@ -9,14 +9,10 @@ properties([parameters([
   booleanParam(defaultValue: false, description: '', name: 'x86_64_win'),
   booleanParam(defaultValue: true, description: 'Build Java bindings', name: 'JavaBindings'),
   choice(choices: 'Release\nDebug', description: 'Java bindings build type', name: 'JBBuildType'),
-  string(defaultValue: 'jp.co.soramitsu.iroha', description: 'Java bindings package name', name: 'JBPackageName'),
-  booleanParam(defaultValue: true, description: 'Build Python bindings', name: 'PythonBindings'),
+  string(defaultValue: 'tech.iroha.libiroha', description: 'Java bindings package name', name: 'JBPackageName'),
```

Comment:
@nickaleks should we really change the namespace or leave it as it was?

Reply:
Not sure about this. Probably we want to leave it as "jp.co.soramitsu.iroha"